deploy/sample-backup-configs/sample_config_build.ini

```diff
@@ -39,6 +41,8 @@ firesim-rocket-quadcore-no-nic-l2-llc4mb-ddr3-half-freq-uncore
 
 firesim-rocket-singlecore-gemmini-no-nic-l2-llc4mb-ddr3
 
+#firesim-rocket-singlecore-small-nvdla-no-nic-l2-llc4mb-ddr3
```

Comment:
Since this config is commented out here and in the `[builds]` section, does that mean users will have to build this AGFI themselves before using it?

Reply:
Yup. That's what I intend.